bin/setup

```diff
@@ -48,19 +48,23 @@ FileUtils.chdir APP_ROOT do
 
   unless File.exist?('.env')
     log "== Setup .env file from .env.example =="
-    cp '.env.example', '.env'
+    system("cp .env.example .env")
```

Comment:
I don't know if there is a better way to do this. This relates to the problems reported by @divagueame and @florianfelsing 

Just one of multiple parts where this is mentioned: https://github.com/rubyforgood/human-essentials/pull/2970#discussion_r898345935

Reply:
Btw, haven't found a reference for the difference between `system!()` and `system()`

Reply:
Does `cp` work for Ubuntu? If not, maybe we can use the `File` method to copy?

Reply:
Ohh woops now I get it, the `system` was missing. Misread at first.